### PR TITLE
feat: add mit license

### LINKED ISSUE #29: Add LICENSE

### PR DESCRIPTION
**Pull request by florian-lefebvre**
Closes #29

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2024 Florian Lefebvre
```
Comment (alexanderniebuhr) [inline LICENSE:3]: I'm fine with having you here first, but I would also like to have the org. What I did in the past was adding either a second line or comma separated e.g. `Florian Lefebvre, Astrolicious Members`

**Review by alexanderniebuhr — APPROVED** [review]
Just a small comment, otherwise looks good..
I wonder if we want to adapt something similar to the `dotdev` repo, which protects the content a little more? _(but we could figure that out later too)_

**Review by TheOtterlord — APPROVED** [review]
same comment as @alexanderniebuhr, but lgtm!

File: LICENSE (hunk 1 above)
Comment (alexanderniebuhr) [reply LICENSE:3]: Just found two more examples in Astro:

https://github.com/withastro/astro/blob/main/LICENSE#L26
https://github.com/withastro/astro/blob/main/LICENSE#L40

Comment (florian-lefebvre) [reply LICENSE:3]: yeah I like the `<author> and Astrolicious contributors`. Will update it! As for being a bit more protective with the content, we can update the license at any time for sure! There are a few topics to discuss first (such as cross posting)

**Pull request merged**